adapters/InMobi/InMobiAdapter/GADMAdapterInMobiBannerAd.m

```diff
@@ -0,0 +1,200 @@
+//
+//  GADMAdapterInMobiBannerAd.m
+//  IMAdMobAdapter
+
+#import "GADMAdapterInMobiBannerAd.h"
+#import <InMobiSDK/IMSdk.h>
+#include <stdatomic.h>
+#import "GADInMobiExtras.h"
+#import "GADMAdapterInMobiConstants.h"
+#import "GADMAdapterInMobiInitializer.h"
+#import "GADMAdapterInMobiUtils.h"
+#import "GADMInMobiConsent.h"
+#import "GADMediationAdapterInMobi.h"
+
+/// Find closest supported ad size from a given ad size.
+static CGSize GADMAdapterInMobiSupportedAdSizeFromGADAdSize(GADAdSize gadAdSize) {
+    // Supported sizes
+    // 320 x 50
+    // 300 x 250
+    // 728 x 90
+    
+    NSArray<NSValue *> *potentialSizeValues =
+    @[ @(GADAdSizeBanner), @(GADAdSizeMediumRectangle), @(GADAdSizeLeaderboard) ];
+    
+    GADAdSize closestSize = GADClosestValidSizeForAdSizes(gadAdSize, potentialSizeValues);
+    return CGSizeFromGADAdSize(closestSize);
+}
+
+@implementation GADMAdapterInMobiBannerAd {
+    
+    id<GADMediationBannerAdEventDelegate> _bannerAdEventDelegate;
+    
+    /// Ad Configuration for the banner ad to be rendered.
+    GADMediationBannerAdConfiguration *_bannerAdConfig;
+    
+    GADMediationBannerLoadCompletionHandler _bannerRenderCompletionHandler;
+    
+    /// InMobi banner ad object.
+    IMBanner *_adView;
+    
+    /// InMobi Placement identifier.
+    NSNumber *_placementIdentifier;
+}
+
+-(instancetype)initWithPlacementIdentifier:(NSNumber *)placementIdentifier {
+    self = [super init];
+    if (self) {
+        _placementIdentifier = placementIdentifier;
+    }
+    return self;
+}
```

Comment:
These lines can be removed? The `_placementIdentifier` instance var. isn't used any where?